### PR TITLE
postgres image build

### PR DESCRIPTION
**Pull request by ChuckHend**
Build and publish docker images with pg_timeseries pre-installed

closes https://github.com/tembo-io/pg_timeseries/issues/16

### CODE REVIEW

**Comment by ChuckHend** [comment]
actions are currently having an issue...https://www.githubstatus.com/

**Comment by ChuckHend** [comment]
> actions are currently having an issue...https://www.githubstatus.com/

I think these are resolved enough to move forward with this PR

File: doc/timeseries.md [hunk 1]
```diff
@@ -7,9 +7,36 @@ The purpose of this extension is to provide a cohesive user experience around th
...
+```
+
+Then connect to the database and enable the extension:
+
+```bash
+psql postgres://postgres:postgres@localhost:5432/postgres
+```
+
+```sql
+CREATE EXTENSION timeseries CASCADE;
```
Comment (jasonmp85) [inline doc/timeseries.md:27]: In the PostgreSQL docker image, I believe there is a folder on disk you can put `sh` and `sql` files in and they'll be run on first startup… maybe `CREATE EXTENSION` could be added there so the user doesn't have to run it?

This is documented [here](https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts). You'd just need to make an `sql` file with `CREATE EXTENSION` in it, then `COPY` it to that folder during the build.

**Review by jasonmp85 — APPROVED** [review]
Approving, but also added one comment about an improvement.

**Comment by jasonmp85** [comment]
Quick question: do we actually know that ARM works? I didn't think Columnar existed in Trunk for that architecture?

**Comment by jasonmp85** [comment]
> Quick question: do we actually know that ARM works? I didn't think Columnar existed in Trunk for that architecture?

Nevermind, I see you're building all the extensions from source!

**Pull request merged**